### PR TITLE
feat: add support for properties like ip_filter

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

adds support for properties like `ip_filter`

## Why this way

to support them

### CODE REVIEW

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
When checking `terraform schema` getting: 
```
"ip_filter": {
              "type": "string",
              "description": "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
              "description_kind": "plain",
              "optional": true
            },
```

And also this in the schema:
```
"ip_filter": {
			Description: "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
			Optional:    true,
			Type:        schema.TypeString,
		},
```

**Comment by Serpentiel** [comment]
> When checking `terraform schema` getting:
> 
> ```
> "ip_filter": {
>               "type": "string",
>               "description": "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
>               "description_kind": "plain",
>               "optional": true
>             },
> ```
> 
> And also this in the schema:
> 
> ```
> "ip_filter": {
> 			Description: "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
> 			Optional:    true,
> 			Type:        schema.TypeString,
> 		},
> ```

this has been resolved, please check again

File: internal/schemautil/user_config/service_types.go [hunk 1]
```diff
@@ -114,6 +143,35 @@ func ServiceTypeCassandra() *schema.Schema {
...
+		"ip_filter": {
+			Description:      "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
+			DiffSuppressFunc: schemautil.IpFilterArrayDiffSuppressFunc,
+			Elem: &schema.Schema{
+				DiffSuppressFunc: schemautil.IpFilterValueDiffSuppressFunc,
+				Type:             schema.TypeString,
+			},
+			MaxItems: 1024,
+			Optional: true,
+			Type:     schema.TypeList,
```
Comment (ivan-savciuc) [inline internal/schemautil/user_config/service_types.go:155]: can we use `schema,TypeSet` instead of list for the ip_filter string

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**